Source/WebCore/platform/graphics/filters/FEBlend.cpp

```diff
@@ -33,9 +33,11 @@
 
 namespace WebCore {
 
-Ref<FEBlend> FEBlend::create(BlendMode mode)
+Ref<FEBlend> FEBlend::create(BlendMode mode, DestinationColorSpace colorSpace)
 {
-    return adoptRef(*new FEBlend(mode));
+    Ref effect = adoptRef(*new FEBlend(mode));
+    effect->setOperatingColorSpace(colorSpace);
```

Comment:
I think we should pass all the parameters into the constructor.  I think we should make a FilterEffect constructor that requires a DestinationColorSpace and sets it in the constructor instead of requiring all the objects to modify after construction.

Reply:
It is a little annoying because setOperatingColorSpace() is a virtual function but I'll look into it.